gapis/shadertools/cc/libmanager.cpp

```diff
@@ -130,11 +130,11 @@ void set_error_msg(code_with_debug_info_t* x, std::string msg) {
   strcpy(x->message, msg.c_str());
 }
 
-std::vector<unsigned int> parseGlslang(const char* code, std::string* err_msg,
-    shader_type type, bool es_profile) {
+std::vector<unsigned int> parseGlslang(const char* code, const char* preamble,
+    std::string* err_msg, shader_type type, bool es_profile, bool relaxed_errs) {
   std::vector<unsigned int> spirv;
 
-  EShMessages messages = EShMsgDefault;
+  EShMessages messages = relaxed_errs ? EShMsgRelaxedErrors : EShMsgDefault;
```

Comment:
Oh, glslang has native support for relaxed compilation?
That makes this less of a hack.  Good!

Reply:
Yeah - I spent a while trying to hack away, then found this. Happy days.